webcompat/static/css/development/components/search-issue.css

```diff
@@ -68,3 +68,7 @@
     align-items: center;
     margin: 3em 0 0 0;
   }
+
+  .wc-SearchIssue-github .wc-Link { 
```

Comment:
IMHO this selector is dangerous because of side effect.

For me, a selector has unique Component (and 2/3 max selector), or can target a simple element.

I prefer `.wc-SearchIssue-github a {}`  or you can write another wrapper aroud the link ``